### PR TITLE
Improve post-module activation notice suggestions.

### PR DESCRIPTION
**Pull request by techanvil**
## Summary

Addresses issue:

- #1148 

## Relevant technical choices

Added NPM dev dependency `@storybook/addon-queryparams` in order to use query params in Storybook stories.

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/sass/components/global/_googlesitekit-publisher-wins.scss [hunk 1]
```diff
@@ -80,6 +88,10 @@
 		}
 	}
 
+	.googlesitekit-publisher-win__image-smaller svg {
```
Comment (asvinb) [inline assets/sass/components/global/_googlesitekit-publisher-wins.scss:91]:
```suggestion
	.googlesitekit-publisher-win__image-smaller {
		svg {
			width: 100%;
		}
	}
```

File: assets/js/components/notifications/BannerNotification/utils.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ export const getContentCellSizeProperties = ( {
  *
  * @since n.e.x.t
  *
- * @param {string} format The format of the notifcation. Can be either small or larger.
+ * @param {string} format The format of the notification. Can be either small or larger.
```
Comment (asvinb) [inline assets/js/components/notifications/BannerNotification/utils.js:74]: Thanks for catching that!

File: assets/js/components/notifications/SetupSuccessBannerNotification.stories.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+export default {
+	title: 'Components/SetupSuccessBannerNotification',
+	component: SetupSuccessBannerNotification,
+	decorators: [
+		withQuery,
+		( Story, { parameters } ) => {
+			const registry = createTestRegistry();
+
+			provideModules( registry, [
```
Comment (asvinb) [inline assets/js/components/notifications/SetupSuccessBannerNotification.stories.js:111]: On top of `provideModules`, we also need to call `provideModuleRegistrations` to provide module registration data to the registry which will fix the issue where the module icons are not rendered.
image
v/s when the icons are displayed:
image

**Review by asvinb — CHANGES_REQUESTED** [review]
@techanvil Nice job! I left some minor comments for you. Can you check them out?
Thanks!

File: assets/sass/components/global/_googlesitekit-publisher-wins.scss (hunk 1 above)
Comment (techanvil) [reply assets/sass/components/global/_googlesitekit-publisher-wins.scss]: Cheers, that's done :+1:

File: assets/js/components/notifications/SetupSuccessBannerNotification.stories.js (hunk 3 above)
Comment (techanvil) [reply assets/js/components/notifications/SetupSuccessBannerNotification.stories.js:112]: Thanks for pointing that out! I've applied the change.

Comment (asvinb) [reply assets/js/components/notifications/SetupSuccessBannerNotification.stories.js:112]: Thanks @techanvil

**Comment by asvinb** [comment]
Thanks @techanvil PR LGTM

**Review by eugene-manuilov — APPROVED** [review]
LGTM

**Pull request merged**